cmd/serve/serve.go

```diff
@@ -56,6 +57,13 @@ func ServeCmd() *cobra.Command {
 			api := api.New(config.API.Size, metrics)
 			aio := aio.New(config.AIO.Size, metrics)
 
+			credentials, err := creds.GetCredentials()
```

Comment:
I believe viper automatically parses configuration from the file (and command line arguments) we shouldn't need to manually parse/set `config.API.Subsystems.Http.Auth` like this